### DISCUSSION TITLE
We are unable to complete your signup. Please read this guide for further details.

### DISCUSSION THREAD

**Discussion by imdeva91** [vercel/community#3976] [Help] [answered]
### Summary

We are unable to complete your signup. Please read [this guide](https://vercel.com/guides/why-can-i-not-signup) for further details.

### Steps to Reproduce

We are unable to complete your signup. Please read [this guide](https://vercel.com/guides/why-can-i-not-signup) for further details.

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble signing up for a new account. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)

**Reply by tonkolviktor** [reply]
so what is the process here? everyone should open a ticket here? Why are you linking a page which says exactly nothing?
